.gitignore

```diff
@@ -1,7 +1,7 @@
 ### ATTENTION ###
```

Comment:
This shouldn't be changed. Please have a look to the correspondent entry in the [wiki](https://github.com/KratosMultiphysics/Kratos/wiki/How-to-make-Git-ignore-the-files-resulting-from-a-compilation-without-conflicts-in-.gitignore)